cassdegrees/ui/views/programs.py

```diff
@@ -14,7 +30,10 @@ def create_program(request):
             return redirect('/list/?view=Degree&msg=Successfully Added Program!')
 
     else:
-        form = EditProgramFormSnippet()
+        if duplicate:
```

Comment:
Wait, this will edit, not duplicate. You need to reset the ID, no?

Reply:
No, I tried it out and it did not edit the existing one. That part of code is for GET requests, and that instance=instance was done just to display the contents of the program user selected to duplicate from.

I have left the POST part of create_program view function as it was before so adding is normal. Attempting to add a duplicate without changing anything will lead to this anyways:
![image](https://user-images.githubusercontent.com/37033052/56935276-4fedcf00-6b33-11e9-8b9c-df3589de54dd.png)